build.rs

```diff
@@ -16,8 +17,11 @@ fn main() {
             .expect("Error during SCSS compilation")
     ).expect("Couldn't write CSS output");
 
+    let cache_id = plume_common::utils::random_hex();
```

Comment:
i'd rather this wasn't random, but deterministic

such as the file's she256sum

Reply:
What is your issue with it being random? It's fixed at compile time, so assuming a production server, which may restart but won't recompile often, the cache will be ok. My first through was using git commit sha, but this may be unnecessary information disclosure

Reply:
if nothing about the code changes, the cache_id shouldn't change either

Reply:
The build won't be run again if the code didn't changed, so the ID will not change.

Reply:
yes, in our development environment that is true, but not for anyone building packages or container images of our software. they always start from a clean slate